condicionales/numeroParOImpar.js

```diff
@@ -0,0 +1,27 @@
+// Escriba un programa que pida primero un número par y luego un número impar (positivos o negativos). En caso de que uno o los dos valores no sea correcto, se mostrará un único aviso.
+const prompt = require("prompt-sync")();
+
+let numberPair;
+let numberOdd;
+
+do {
+    numberPair = parseInt(prompt("Enter a number Pair: "));
+  
+    if (isNaN(numberPair)) {
+      console.log("Error: Enter a valid number!");
+    }
+  } while (isNaN(numberPair));
```

Comment:
I think the exercise order doesn't ask to do a cycle, can you check this please?